### ISSUE TITLE
I have Some Error when add Button to my app ?!!

### ISSUE THREAD

**Issue by anastely** [facebook/react-native#21757]
**i have some issue when adding  just Button !!**

and in the app thats the error

" java.lang.string cannot be cast to com.facebook.react.uimanager.accessibility DelegateUtil$accessibilityRole "

[https://i.stack.imgur.com/n1WuD.png](url)

---
my simple code || 
`

     import React, { Component } from "react";
     import { StyleSheet, TextInput, View, Button, Text } from "react-native";

     export default class App extends Component {
           state = {
                placeName: ""
               };
           placeNameChangeHandler = val => {
                    this.setState({
                   placeName: val
              });
           };
           onPressLearnMore = () => {
                 alert("Pressed");
               };

          render() {
              return (
                    <View style={styles.container}>
                           <Button
                                onPress={this.onPressLearnMore}
                                title="Learn More"
                                color="#841584"    
                            />

                            <TextInput
                                 style={{
                                            width: 300,
                                            borderBottomWidth: 1,
                                            borderBottomColor: "#333"
                                      }}
                                  placeholder="Enter Name.."
                                  value={this.state.placeName}
                                  onChangeText={this.placeNameChangeHandler}
                             />
                 </View>
              );
         }

`
